cluster/common.sh

```diff
@@ -900,14 +900,14 @@ function sha1sum-file() {
   fi
 }
 
-# Downloads cfssl into ${KUBE_TEMP}/cfssl directory
+# Downloads cfssl into $1 directory
 #
 # Assumed vars:
-#   KUBE_TEMP: temporary directory
+#   $1 (cfssl directory)
 #
 function download-cfssl {
-  mkdir -p "${KUBE_TEMP}/cfssl"
-  pushd "${KUBE_TEMP}/cfssl"
+  mkdir -p "$1"
```

Comment:
Can you please a local variable with a meaningful name for "$1" (like [here](https://github.com/kubernetes/kubernetes/pull/42994/files#diff-0a87f271b16b78bb2c6e66cd459d1135R1063))?